posthog/queries/trends/formula.py

```diff
@@ -3,7 +3,7 @@
 from typing import Any, Dict, List
 
 from ee.clickhouse.queries.breakdown_props import get_breakdown_cohort_name
-from ee.clickhouse.sql.clickhouse import trim_quotes_expr
+from posthog.clickhouse.kafka_engine import trim_quotes_expr
```

Comment:
Q: How did this util end up in `kafka_engine` ?

Reply:
It was always there but the naming of the file was just "clickhouse" and I renamed it to be more accurate